calendar/calendar-tool/tool/src/webapp/vm/calendar/calendar_macros.vm

```diff
@@ -0,0 +1,37 @@
+## Macros for Calendar ##
+
+## Macro to create the Calendar View picker (to switch between Day, Week, Month, Year, List calendar views)
+#macro(calendarView)
+			<!-- drop down menu for different views -->
+			<form name="viewForm" method="post" action="#toolForm("CalendarAction")">
```

Comment:
Can you reduce the indentation here? The macro you have below has much less rightward drift.

Reply:
Sure, I'll take a look. Thanks.

Reply:
That indent was based on the HTML markup indenting of where that macro gets inserted, but you're right that it makes it odd and difficult when you are editing that macros file. I'll remove the excess.

Reply:
I've removed the indentation.